### ISSUE TITLE
DELETE on _meta

### ISSUE THREAD

**Issue by awlayton** [OADA/oada-srvc-docker#32]
Are we supposed to be able to delete from meta? Because when I do one I get 204 but the stuff is still there

**Comment by abalmos** [comment]
@awlayton @sanoel  As a follow up ... When I delete a deep key under `_meta` everything in that parent object is deleted and replaced with null.

`GET /resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta`
```json
{
  "_changes": {
    "_id": "resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta/_changes",
    "_rev": 19
  },
  "_id": "resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta",
  "_owner": "users/default:users_sam_321",
  "_rev": 19,
  "_type": "application/vnd.trellisfw.audit.globalgap.1+json",
  "modified": 1579833483.075,
  "modifiedBy": "users/default:users_sam_321",
  "services": {
    "trellis-signer": {
      "tasks": {
        "e6859766-93e6-43ae-b8ba-886be752840b": {
          "test-task-id": {
            "status": "pending"
          }
        },
        "507f07d1-cd3b-4c88-8e66-e4a87175e183": {
          "status": "pending"
        }
      }
    }
  },
  "stats": {
    "createdBy": "users/default:users_sam_321",
    "created": 1579727484.483
  }
}
```

Followed by `DELETE https://smithfield.trellis.one/resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta/services/trellis-signer/tasks/507f07d1-cd3b-4c88-8e66-e4a87175e183`

Results in
`GET /resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta`
```json
{
  "_changes": {
    "_id": "resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta/_changes",
    "_rev": 19
  },
  "_id": "resources/f2d8a27d-19d4-4b60-8a84-82bae137c990/_meta",
  "_owner": "users/default:users_sam_321",
  "_rev": 19,
  "_type": "application/vnd.trellisfw.audit.globalgap.1+json",
  "modified": 1579833483.075,
  "modifiedBy": "users/default:users_sam_321",
  "services": null,
  "stats": {
    "createdBy": "users/default:users_sam_321",
    "created": 1579727484.483
  }
}
```

**Comment by awlayton** [comment]
Weird. When I did it nothing got changed....

**Comment by awlayton** [comment]
I haven't really figured out what was going on here...

**But** it *might* be fixed by ca135d9f0e2899f4ef759b5b117ea2b79fd43852 as well, especially since aside from that there shouldn't be anything special about how `_meta` is treated here.

**Comment by awlayton** [comment]
This _might_ be fixed by #33

**Comment by awlayton** [comment]
Fixed in #33
